chapter-04/Self Driving Car - Behavior Cloning.ipynb

```diff
@@ -0,0 +1,1763 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import tensorflow as tf\n",
+    "import matplotlib.pyplot as plt\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from tensorflow.contrib.learn.python.learn import monitors"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load the datasets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "FILE_PATH = \"C:\\\\Users\\\\quanh\\\\Desktop\\\\datasets\\\\06\\\\driving_log.csv\"\n",
```

Comment:
I think it'd be cleaner to use relative paths both here and in the book.

Reply:
Sure. I will change it.